tests/tensorflow/test_ignored_scopes.py

```diff
@@ -43,8 +43,8 @@ def test_ignored_scopes():
     compressed_model, _ = create_compressed_model_and_algo_for_test(model, config, force_no_init=True)
 
     ref_fake_quantize_names = [
-        'conv1_input/fake_quantize',
-        'some_scope/conv3/c2/fake_quantize',
+        'some_scope/conv2/c1/fake_quantize',
+        'some_scope/conv3/c3/fake_quantize',
```

Comment:
Could you explain why the FQ names have changed?

Reply:
![ignored_scopes](https://user-images.githubusercontent.com/31695470/125283704-55551f80-e321-11eb-85e3-55d8de230029.png)
Here is the quantizer propagation visualization. The nodes whose names matched against the ignored scopes do not have quantized inputs and do not allow quantizers to be propagated from driectly below them into the positions above. Note that the ignored scope in the test has both global and activation-wise settings. I guess that either these weren't correctly handled previously, or the quantizer propagation itself has impacted the ignored scopes meaning (before the ignored scopes had to relate to where the quantizer ended up, and now the ignored scopes rather decides which node will be quantized in the beginning of the propagation and where they can propagate).